pandas/core/apply.py

```diff
@@ -414,17 +418,25 @@ def maybe_apply_str(self) -> Optional[FrameOrSeriesUnion]:
         f = self.f
         if not isinstance(f, str):
             return None
+
+        obj = self.obj
+
+        if f == "size" and isinstance(obj, ABCDataFrame):
```

Comment:
doesn't this affect Series as well?

Reply:
For a Series, size gives the number of rows and so the default path calling `obj.size` produces the correct result.